lib/components/PathProgress/PathProgress.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { cx } from '../../cva';
+
+export interface PathProgressProps {
+  steps: number;
+  currentStep: number;
+}
+
+const getButtonClasses = (active: boolean, idx: number, isIcon = false) =>
+  cx('size-8 rounded-full appearance-none z-10 mb-6', {
+    'bg-accent !text-white': active,
```

Comment:
`!text-white`
Would be nice not to use `!important` in design system.

One way might be to have own class for the actual text/icon and separate the logic from the background.

Also the `!mb-0` would be nice to done without `!important`

Reply:
I was wondering why the classes don't get applied without important, and the culprit was `cx` combined with `twMerge`. They would omit `text-white` if `text-button-lg` was used. Also the order of the object properties in `cx` also affected the inclusion of css classes.